projects/components/src/confirmation/confirmation.module.ts

```diff
@@ -1,10 +1,11 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
+import { FormattingModule } from '@hypertrace/common';
 import { ButtonModule } from '../button/button.module';
 import { ConfirmationModalComponent } from './confirmation-modal.component';
 
 @NgModule({
   declarations: [ConfirmationModalComponent],
-  imports: [CommonModule, ButtonModule]
+  imports: [CommonModule, ButtonModule, FormattingModule]
```

Comment:
The confirmation modal changes I'm guessing are happening separately (which makes sense), but allows us to roll back the changes to those files.